ee/tabby-webserver/src/repositories/mod.rs

```diff
@@ -12,42 +12,40 @@ use axum::{
 };
 use hyper::Body;
 use juniper_axum::extract::AuthBearer;
+pub use resolve::RepositoryCache;
 use tabby_common::config::RepositoryConfig;
 use tracing::{instrument, warn};
 
 use crate::{
-    repositories::resolve::{
-        contains_meta, resolve_all, resolve_dir, resolve_file, resolve_meta, RepositoryMeta,
-        ResolveParams,
-    },
+    repositories::resolve::{RepositoryMeta, ResolveParams},
     schema::auth::AuthenticationService,
 };
 
 #[derive(Debug)]
 pub struct ResolveState {
```

Comment:
These do have different contexts for where they are used, though right now it contains nothing else. I think for now I'd prefer to leave it as a type alias so that we have an opportunity to easily convert it back into a struct and add more fields if needed in the future.